vendor/k8s.io/cloud-provider/features/gce.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
do we need this code? run `dep ensure -update`

Reply:
no, we don't. it is coming because of  indirect dependency in this vendor file `k8s.io/kubernetes/pkg/features`